tests/history/test_db.py

```diff
@@ -1,19 +1,19 @@
 import datetime
-from aiohttp.test_utils import make_mocked_coro
 
 import pytest
+from aiohttp.test_utils import make_mocked_coro
 
 import virtool.history.db
 
 
 class TestAdd:
 
-    async def test(self, snapshot, dbi, static_time, test_otu_edit, test_change, tmp_path):
+    async def test(self, snapshot, dbi, static_time, test_otu_edit, test_change, tmp_path, mock_config):
+        config = mock_config
```

Comment:
See, what is the point of calling the fixture `mock_config`?

More useable to just call it `config`. We know it is a `pytest` fixture because it is an argument in a test method.